### ISSUE TITLE
Breadcrumb link accessibility

### ISSUE THREAD

**Issue by jamie-lovell** [alphagov/govuk_frontend_toolkit#460]
Hi,
The breadcrumb <a> tag currently doesn't change colour in its hover state. In an app that we're building using breadcrumbs, it's failing accessibility because of this. Considering the link is styled black with an underline - what would the guidance be for the colour to change to on hover? Happy to make the change in a PR.

**Comment by 36degrees** [comment]
Hi Jamie,

I am not aware of any requirement that says links must have a distinct hover state, as long as it's clear that they are clickable. It would be good to understand what you mean when you say it's failing accessibility – is this something that's been flagged in an audit or by automated testing tools?

Thanks,

Ollie

**Comment by 36degrees** [comment]
Following the launch of the [GOV.UK Design System](https://design-system.service.gov.uk/), GOV.UK Frontend Toolkit will now only get major bug fixes and security patches, so I'm going to close this.

The behaviour of the breadcrumbs is the same in GOV.UK Design System / Frontend as it is in Elements, so if you believe this is still an issue please add it to the [backlog issue for Breadcrumbs](https://github.com/alphagov/govuk-design-system-backlog/issues/33).
